packages/core/src/components/MonthPicker/MonthPicker.scss

```diff
@@ -0,0 +1,17 @@
+@import '@cmsgov/design-system-support/src/index';
+
+/*
+Month Picker
+
+@uswds https://standards.usa.gov/components/form-controls
+
```

Comment:
Let's add an HTML example? We could leave out the "Select/Clear all" buttons though since those require JS.